libs/external-data/src/lib/airtable/types.ts

```diff
@@ -59,6 +59,25 @@ export interface UxTestData {
   description?: string;
 }
 
+export interface AnnotationsData {
+  airtable_id: string;
+  title: string;
+  title_fr: string;
+  event_type?: string[];
```

Comment:
```suggestion
  event_type: AnnotationsEventType[];
```
Should be required, plus type alias